dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -437,6 +437,8 @@ returnToPrimaryServer.desc=A boolean value that indicates if the search should b
 timestampFormat=Timestamp format
 timestampFormat.desc=A string value that provides a SimpleDateFormat pattern that is used to parse timestamp attribute values. For example, you can use 'yyyyMMddHHmmss.SSSZ' to parse '20181120214852.869-0000Z'. If this attribute is not defined, a default will be provided based on ldapType.
 
+allowWriteToSecondaryServers=Allow write to secondary servers
+allowWriteToSecondaryServers.desc=Allow creates, updates, and deletes on failover servers.
```

Comment:
suggested edit:

Allow create, update, and delete operations on failover servers.